### PR TITLE
AC Test Fix - LRAC-9599 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://testray.liferay.com/home/-/testray/case_results?tab=runs&testrayBuildId=1612297766 (run)
https://issues.liferay.com/browse/LRAC-9599 (ticket)

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase [hunk 1]
```diff
@@ -292,8 +277,8 @@ definition {
 			ProxyUtil.startHarRecording("pageDepthReached");
 		}
 
-		task ("Scroll down by 200") {
-			ScrollBy(value1 = "0,200");
+		task ("Scroll down by 4000") {
+			ScrollBy(value1 = "0, 4000");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase:281]: Hey @alexcandidos, I was thinking about this space here, do you think we should leave it that way or does this cause any questions for anyone who will use it in the future?

@AfonsoHSantos said it works this way.

Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase:281]: hi @ViniLopes20 , looking at the liferay-portal repo, in most of the cases there is a blank space between the parameters, actually, only in this test case it was missing, so I think it is how it should be.

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase:281]: Ohhh ok, thanks @alexcandidos

**Review by ViniLopes20 — APPROVED** [review]

**Comment by alexcandidos** [comment]
Hi @AfonsoHSantos , awesome work my Doctor!!! ⭐ 

Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**